### PR TITLE
[jsoncons] Update to v0.154.1

### PR DESCRIPTION
**Pull request by danielaparker**
**Describe the pull request**

Update to v0.154.1
Includes enhancements #257, #204

- What does your PR fix? Fixes #

#256
#255
#259

- Which triplets are supported/not supported? Have you updated the CI baseline?

All are supported. Baseline has been updated.

- Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?

Yes

### CODE REVIEW

**Review by LilyWangL — APPROVED** [review]

File: ports/jsoncons/CONTROL [hunk 1]
```diff
@@ -1,4 +1,4 @@
 Source: jsoncons
-Version: 0.153.3
-Description: A C++, header-only library for constructing JSON and JSON-like text and binary data formats, with JSON Pointer, JSON Patch, JSONPath, CSV, MessagePack, CBOR, BSON, UBJSON
+Version: v0.154.1
```
Comment (ras0219-msft) [inline ports/jsoncons/CONTROL:2]: We prefer to remove `v` prefixes from the `Version:` field

**Review by ras0219-msft — CHANGES_REQUESTED** [review]
LGTM to merge once `v` is removed

File: ports/jsoncons/CONTROL (hunk 1 above)
Comment (danielaparker) [reply ports/jsoncons/CONTROL]: v prefix has been removed in CONTROL file

**Comment by danielaparker** [comment]
Just FYI, v prefix has been removed.
Thanks,Daniel
    On Monday, July 13, 2020, 03:16:51 p.m. EDT, Robert Schumacher <notifications@github.com> wrote:  
 
 


@ras0219-msft requested changes on this pull request.

LGTM to merge once v is removed

In ports/jsoncons/CONTROL:
> @@ -1,4 +1,4 @@
 Source: jsoncons
-Version: 0.153.3
-Description: A C++, header-only library for constructing JSON and JSON-like text and binary data formats, with JSON Pointer, JSON Patch, JSONPath, CSV, MessagePack, CBOR, BSON, UBJSON
+Version: v0.154.1

We prefer to remove v prefixes from the Version: field

**Review by ras0219 — APPROVED** [review]

**Comment by strega-nil** [comment]
Cool, LGTM! Thanks @danielaparker :)

**Pull request merged**